src/qt/rpcconsole.cpp

```diff
@@ -892,8 +892,6 @@ void RPCConsole::banSelectedNode(int bantime)
         SplitHostPort(nStr, port, addr);
 
         CNode::Ban(CNetAddr(addr), BanReasonManuallyAdded, bantime);
-        bannedNode->fDisconnect = true;
```

Comment:
The variable `bannedNode` seems to be unused here now.

The similar code is a few lines above in `RPCConsole::disconnectSelectedNode()` - same changes needed?